modules/installation-vsphere-installer-network-requirements.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-installer-network-requirements_{context}"]
+= VMware vSphere network connectivity requirements
```

Comment:
Suggest to keep consistence as one described in upi section. In upi section, it states "Network connectivity requirements".